front/src/pages/settings/SettingsObjects.tsx

```diff
@@ -52,6 +55,11 @@ const StyledIconDotsVertical = styled(IconDotsVertical)`
   color: ${({ theme }) => theme.font.color.tertiary};
 `;
 
+const StyledHeader = styled.div`
+  display: flex;
```

Comment:
Hi @Abhi3685, thank you for your contribution!

On Figma, the title and the button are vertically center aligned, and the header has a 32px margin-bottom. To achieve this we could:
- override the `H1Title` margin-bottom (add a `className` prop to the `H1Title` component and then style it with emotion to add `margin-bottom: 0`).
- add a 32px (`theme.spacing(8)`) margin-bottom to the `StyledHeader`
- use align-items on the `StyledHeader`

Otherwise, LGTM!